js/ui/drop_down_button.js

```diff
@@ -398,8 +398,8 @@ let DropDownButton = Widget.inherit({
             focusStateEnabled: this.option("focusStateEnabled"),
             hoverStateEnabled: this.option("hoverStateEnabled"),
             onItemClick: this._buttonGroupItemClick.bind(this),
-            width: this.option("width"),
-            height: this.option("height"),
+            width: "100%",
```

Comment:
Why this width is 100% but width of the popup is a function?

Reply:
Cause "dx-overlay-wrapper dx-popup-wrapper" element isn't in the "dx-dropdownbutton", but "dx-buttongroup" is. Overlay wrapper stretches across the entire width of the screen, not across dropDownButton widget.